iothub/device/src/Serialization/DefaultPayloadConvention.cs

```diff
@@ -1,19 +1,40 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
 namespace Microsoft.Azure.Devices.Client
 {
     /// <summary>
-    /// The default implementation of the <see cref="PayloadConvention"/> class.
+    /// The default implementation of the <see cref="PayloadConvention"/> class for Newtonsoft.Json.
     /// </summary>
     /// <remarks>
-    /// This class makes use of the <see cref="NewtonsoftJsonPayloadSerializer"/> serializer and the <see cref="Utf8PayloadEncoder"/>.
+    /// This class makes use of the <see cref="JsonConvert"/> serializer and the <see cref="Encoding.UTF8"/> encoder.
     /// </remarks>
     public sealed class DefaultPayloadConvention : PayloadConvention
     {
-        private DefaultPayloadConvention()
+        private static readonly JsonSerializer s_jsonSerializer = new();
+
+        /// <summary>
+        /// A static instance of JsonSerializerSettings which sets DateParseHandling to None.
+        /// </summary>
+        /// <remarks>
+        /// By default, serializing/deserializing with Newtonsoft.Json will try to parse date-formatted
+        /// strings to a date type, which drops trailing zeros in the microseconds portion. By
+        /// specifying DateParseHandling with None, the original string will be read as-is. For more details
+        /// about the known issue, see https://github.com/JamesNK/Newtonsoft.Json/issues/1511.
+        /// </remarks>
+        private static readonly JsonSerializerSettings s_settings = new()
```

Comment:
Imports all the relevant functionality from the NewtonsoftJsonPayloadSerializer and Utf8PayloadEncoder types.